Source/WebCore/page/Page.cpp

```diff
@@ -4885,6 +4885,14 @@ void Page::performOpportunisticallyScheduledTasks(MonotonicTime deadline)
     if (m_opportunisticTaskScheduler->hasImminentlyScheduledWork())
         options.add(JSC::VM::SchedulerOptions::HasImminentlyScheduledWork);
     commonVM().performOpportunisticallyScheduledTasks(deadline, options);
+
```

Comment:
LGTM
maybe we should add a comment here?

Reply:
Agreed, or wrapping the below lines in a well named function.